src/app/list/basic-list/examples/list-basic-example.component.ts

```diff
@@ -172,7 +172,7 @@ export class ListBasicExampleComponent implements OnInit {
    * @returns {ActionConfig}
    */
   getActionConfig(item: any, actionButtonTemplate: TemplateRef<any>,
-      startButtonTemplate: TemplateRef<any>): ActionConfig {
+    startButtonTemplate: TemplateRef<any>): ActionConfig {
```

Comment:
Same here. The line wrapping indentation is removed.

Reply:
This one is a bit more complicated for me. I think it should either be a normal 2 space indent or it is indented to be inline with the same context above. That would be about 17-18 space indent. I have done both but not an arbitrary 4 space indent. But that is just my experience.

Reply:
It's a Java style convention that I've always followed. It's my preference, but can live without it.